app/static/css/main.css

```diff
@@ -2611,3 +2611,39 @@ article.text-post.comment.deleted {
   font-weight: bold;
   text-align: center;
 }
+
+.admin-config {
+  table-layout: fixed;
+  width: 95%;
+}
+
+.admin-config-name {
+  width: 35%;
+}
+
+.admin-config-value {
+  width: 60%;
+}
+
+.admin-config-action {
+  width: 5%;
+}
+
+.admin-config-doc-toggle {
+  font-size: smallest;
+  padding-right: 5px;
+}
+
+.admin-config-edit-form input {
+  width: 100%;
+}
+
+.admin-config-edit {
+  fill:  #00419d;
+  width: 16px;
+  height: 16px;
+}
```

Comment:
`admin-config-doc-toggle` and `admin-config-edit` should have `cursor: pointer` so it's easier to realize that it's clickable

Reply:
Done.